### PR TITLE
Deployment

### PR DESCRIPTION
**Pull request by Tommymarvel**
I have work on a personal portfolio project setting up the files correctly and making it work perfectly for deployment.

### CODE REVIEW

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, PLease provide a link to your deployed app on the readme 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Tommymarvel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

- [ ] You need to add the URL for the deployed website in your Readme file. I can see that you have deployed the site already, but you have not provided a `live link` in the readme file.

- [ ] After you have added the link to your Readme file make sure the site works as expected and you will be good to go.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+## Getting Started
+
+**Instructions on how to access my project locally.**
+ To Access my project locally please follow the following instructions;
+1. Enter this url:  https://github.com/Tommymarvel/Tommymarvel.github.io in your web browser.
+2. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side.
+3. Select download Zip option from drop down menu.
+4. Once the download is complete you will be able to access my project locally.
+Here is my repository link: https://github.com/Tommymarvel/Tommymarvel.github.io
```
Comment (vechicin) [inline README.md:24]: These instructions are incorrect. The link you added in this section leads to a README that has a link to your live demo. If you plan on providing instructions on how to copy the project locally, please make sure they are correct.

File: README.md [hunk 2]
```diff
@@ -1 +1,53 @@
...
+
+
+## Authors
+
+👤 **Tommy Marvel**
+- GitHub: [https://github.com/tommymarvel)
+- Twitter: [@tomiwa_ibironke](https://twitter.com/tomiwa_ibironke) 
+- Go [**LIVE**](https://tommymarvel.github.io/)
```
Comment (vechicin) [inline README.md:32]: Make sure you link your GitHub profile properly and add your live demo link under a `Live Demo` header, just like in the template for this README.

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (vechicin) [inline README.md:49]: If you plan on keeping this section, make sure you give some actual acknowledgments. You can mention Microverse and/or the Figma template here.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Tommymarvel ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job trying to implement the previous reviewer's suggestions

 ## Required Changes ♻️

- [ ] This is how your demo's mobile version looks on my end.

Screen Shot 2022-06-16 at 7 24 30

Please fix it as the `Headline` section doesn't look like the one from the template design.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @Tommymarvel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**